docs/Developers/Review.rst

```diff
@@ -72,6 +72,11 @@ The review process for coala is as follows:
     - ``reack commit_SHA`` in case the commit was acknowledged before, was
       rebased without conflicts and the rebase did not introduce logical
       problems.
+
+    .. note::
+
+        Only one acknowledgment is needed per commit i.e. ack <commit-sha>.
```

Comment:
I think it would be better if the same placeholder was used for the commit-sha as in the rest of the document. Instead of <commit-sha> we could have commit_SHA (within the quotes ` ), as in the rest of the file to maintain uniformity